testbed/testbed/mock_backend_test.go

```diff
@@ -49,6 +49,35 @@ func TestGeneratorAndBackend(t *testing.T) {
 	assert.Equal(t, lg.DataItemsSent(), mb.DataItemsReceived())
 }
 
+func TestZipkinGeneratorAndBackend(t *testing.T) {
```

Comment:
Instead of creating this func by copying TestGeneratorAndBackend it would be nicer to change TestGeneratorAndBackend to use subtests and make Jaeger and Zipkin the 2 subtests.